main.cpp

```diff
@@ -14,11 +14,47 @@ Create a branch named Part2
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person	
+{	
+	int age;
+	int height;
+	float hairLength;
+	float GPA;
+	unsigned int SATScore;
 
+    struct Foot				
+	{			
+        float distanceFromGround = 0 ;
+        float momentum = 0;
 
+    	int stepForward( float momentum, float distanceFromGround );		
+		float stepSize();		
+	};			
```

Comment:
- [ ] fix this formatting.

Reply:
I fixed the spaces around "(" . 
The indentations looks good on replit. I just deleted the tabs and added them again.